issue9/src/projects.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Projects</title>
+</head>
+<body class="project-page flex-container">
+    <main class="page-content flex-container">
+        <div class="grid-container grid-container--8x7 grid-container--12x4 project-pg-hero">
+            <header class="page-header">
+                <nav class="nav-bar">
+                    <div class="logo-container">
+                        <a href="index.html">Site Logo</a>
+                    </div>
+                    <div class="nav-btn-container">
+                        <a href="about.html" class="nav-btn-container__link about-page">About</a>
+                        <a href="projects.html" class="nav-btn-container__link project-page">Projects</a>
+                        <a href="contact.html" class="nav-btn-container__link contact-page">Contact</a>
+                    </div>
+                </nav>
+            </header>
+            <!--Projects Summary Section-->
+            <section class="grid-item grid-item-project grid-item-project-1">
+                <div class="project-img-container">
+                    <p>Project image will go here</p>
+                </div>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```

Comment:
I'm not a fan that you're still using `lorem ipsum`. I believe you had a design that had content. We shouldn't be 4 weeks after that design still doing the same stuff. Ur page does look close-ish to your design. However, I'm also not a fan of the copy paste folders `issue9`. I'd rather you just had 1 website that you were consistently working on instead of pushing to different weeks. I added to the Feedback doc that there needs to be a better explanation around how the folder structure should work. Another thing about adding the folders for each week is that this looks like a 33 file change, where you're really only updating 2 files it seems. That makes it harder on the code reviewers to review.

Reply:
I updated the design, added content and made a logo.